diff --git a/dist/lib/src/compiler-path.js b/dist/lib/src/compiler-path.js
index 72a74f45332fab89e3bdd09c9a7214ae8a88948c..e5818988c0a64563e5a360bf9d5979eabcf92b2b 100644
--- a/dist/lib/src/compiler-path.js
+++ b/dist/lib/src/compiler-path.js
@@ -16,6 +16,8 @@ const isLinuxMusl = function () {
 };
 /** The full command for the embedded compiler executable. */
 exports.compilerCommand = (() => {
+    return ['sass']
+
     const platform = process.platform === 'linux' && isLinuxMusl()
         ? 'linux-musl'
         : process.platform;
